notebooks/Roberts_Mock_data.ipynb

```diff
@@ -0,0 +1,1113 @@
+              "\n",
+              "        async function convertToInteractive(key) {\n",
+              "          const element = document.querySelector('#df-1f1f5b66-0be2-496d-93d4-a268dee393d7');\n",
+              "          const dataTable =\n",
+              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
+              "                                                     [key], {});\n",
+              "          if (!dataTable) return;\n",
+              "\n",
+              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
+              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
+              "            + ' to learn more about interactive tables.';\n",
+              "          element.innerHTML = '';\n",
+              "          dataTable['output_type'] = 'display_data';\n",
+              "          await google.colab.output.renderOutput(dataTable, element);\n",
+              "          const docLink = document.createElement('div');\n",
+              "          docLink.innerHTML = docLinkHtml;\n",
+              "          element.appendChild(docLink);\n",
+              "        }\n",
+              "      </script>\n",
+              "    </div>\n",
+              "  </div>\n",
+              "  "
+            ],
+            "text/plain": [
+              "             profile_id  ...                   availability\n",
+              "0    O67192423422555833  ...                      [Morning]\n",
+              "1     O2910554661409564  ...                    [Afternoon]\n",
+              "2    O43116830919747129  ...  [Afternoon, Evening, Morning]\n",
+              "3    O49680502635482637  ...  [Afternoon, Evening, Morning]\n",
+              "4    O57424789435443509  ...                      [Morning]\n",
+              "..                  ...  ...                            ...\n",
+              "195  O22173446303508952  ...                    [Afternoon]\n",
+              "196  O40448878781702092  ...                      [Morning]\n",
+              "197  O64803161155063127  ...                    [Afternoon]\n",
+              "198  O51328340886255708  ...                      [Evening]\n",
+              "199  O52028868511979280  ...  [Afternoon, Evening, Morning]\n",
+              "\n",
+              "[200 rows x 5 columns]"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 223
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "##Total Number of Mentees\n",
+        "\n",
+        "The total number of generated mentees is going to be based off the total sum of mentee_load in the mentor DataFrame. I have a feeling this will give us a surplus of mentees, especially after considering all the pertinent features. \n",
```

Comment:
I really like this feature: I think there is more to explore here. But it does seem that a hard limit on the number of mentee's the program could account for would be the number of mentees that the mentors are willing to take on.

Reply:
Yeah, this one was odd for me because if we want to give the mentor the most control over how they schedule and what not, but give a limit to their time because they can't take on all the mentees. I'm open to ideas though.